perf_dashboard/deploy/perf-dashboard/Chart.lock

```diff
@@ -0,0 +1,6 @@
+dependencies:
+- name: metrics-server
```

Comment:
Why do we need this? GKE already has this deployed I think?

Reply:
If that is the case, then we can remove this.